documentation/source/users/rmg/input.rst

```diff
@@ -359,17 +359,55 @@ As a contrast, a typical set of parameters for non-pruning is::
         toleranceInterruptSimulation=0.5,
     )
 
-where ``toleranceKeepInEdge`` is always 0, meaning all the edge species will be kept in edge since all the edge species have positive flux. 
-``toleranceInterruptSimulation`` equals to ``toleranceMoveToCore`` so that ODE simulation get interrupted once discovering a new core species. 
+where ``toleranceKeepInEdge`` is always 0, meaning all the edge species will be kept in edge since all the edge species have positive flux.
+``toleranceInterruptSimulation`` equals to ``toleranceMoveToCore`` so that ODE simulation get interrupted once discovering a new core species.
 Because the ODE simulation is always interrupted, no pruning is performed.
 
 Please find more details about the theory behind pruning at :ref:`Pruning Theory <prune>`.
 
+Advanced Setting: Thermodynamic Pruning
+----------------------------------------------------
+Thermodynamic pruning is an alternative to flux pruning that does not require a given
+simulation to complete to remove excess species.  The thermodynamic criteria is
+calculated by determining the minimum and maximum Gibbs energies of formation (Gmin and Gmax)
+among species in the core.  If the Gibbs energy of formation of a given species is G
+the value of the criteria is (G-Gmax)/(Gmax-Gmin).  All of the Gibbs energies are evaluated
+at the highest temperature used in all of the reactor systems.  This means that a value of 0.2
+for the criterion implies that it will not add species that have Gibbs energies of formation
+greater than 20% of the core Gibbs energy range greater than the maximum Gibbs energy of
+formation within the core.
+
+For example ::
+
+    model(
+        toleranceMoveToCore=0.5,
+        toleranceInterruptSimulation=0.5,
+        toleranceThermoKeepSpeciesInEdge=0.5,
+        maximumEdgeSpecies=200000,
```

Comment:
I'm afraid I don't understand, although I'm a little confused by what you mean by "is required if only thermodynamic based pruning is used".  Why might there be confusion about that particular parameter, but not about any of the other parameters?  I made the model block so that it would include all parameters relevant to a thermodynamic pruning run and I specifically mention the maximumEdgeSpecies parameter in the next paragraph.